### PR TITLE
Updated python.rst and Created example_taskflow_sensor.py

### PR DESCRIPTION
**Pull request by rohan472000**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by ankit-gautam23 — APPROVED** [review]
LGTM

**Comment by rohan472000** [comment]
@potiuk @ankit-gautam23 please check static checks failure, and suggest any rectification as I'm not getting how to resolve that.

**Comment by ankit-gautam23** [comment]
Mark us as reviewers, we will see

**Comment by rohan472000** [comment]
> Mark us as reviewers, we will see

You are already a reviewer for this PR, check in Reviewers.

File: airflow/example_dags/example_taskflow_sensor.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+with DAG(
+    "example_dag",
+    default_args=default_args,
+    description="Example DAG with PythonSensor",
+    schedule_interval=None,
+    catchup=False,
+) as dag:
+    # [START example_taskflow_sensor]
+    t1 = PythonSensor(
```
Comment (pankajastro) [inline airflow/example_dags/example_taskflow_sensor.py:43]: - example sensor with task flow API https://airflow.apache.org/docs/apache-airflow/stable/tutorial/taskflow.html#using-the-taskflow-api-with-sensor-operators
- PythonSensor example https://airflow.apache.org/docs/apache-airflow/stable/howto/operator/python.html#pythonsensor

so, not sure if we need this new example.

Comment (rohan472000) [reply airflow/example_dags/example_taskflow_sensor.py:43]: so what rectification you would  suggest??

**Pull request closed without merging**